### PR TITLE
LPS-0 DO NOT MERGE Run all forms tests

### PR DESCRIPTION
**Pull request by pedroqueiroz**

### CODE REVIEW

**Comment by pedroqueiroz** [comment]
ci:test

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java [hunk 1]
```diff
@@ -136,6 +141,21 @@ public int getAutosaveInterval() {
 		return _ddmFormWebConfiguration.autosaveInterval();
 	}
 
+	public Map<String, String> getAvailableExportFormats() {
```
Comment (rafaprax) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java]: Hey @pedroqueiroz, I think this method should be part of the `DDMExporterFactory` class. What do you think?

**Comment by pedroqueiroz** [comment]
ci:test

**Comment by pedroqueiroz** [comment]
@rafaprax , modifications done and tests added. :)

**Comment by rafaprax** [comment]
@pedroqueiroz conpile error. Please fix and resend

**Pull request closed without merging**